pom.xml

```diff
@@ -605,6 +605,21 @@
       <version>3.5.1</version>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
There dependencies are WAY too old. They need to be replaced with something newer..